ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -694,7 +694,7 @@ namespace
         {
             if (box_encoding == V5BoxEncoding::CORNER)
             {
-                normalize_corner(boxes, boxes_shape);
+                // normalize_corner(boxes, boxes_shape);
```

Comment:
Why is this necessary? Or what happens if you don't do this? I'm concerned about whether this is implementing a necessary feature that we are unaware of.

Reply:
Yes, the reason to close and not implement it on our side is that it changes one box to another box so not meet the user's request.  

If we have a box which coordinate ([y1, x1, y2, x2]) is [1, 3, 4, 2] ,  the function will make it to [1, 2, 4, 3],  these two boxes are not equivalent, this means if we link two op, the output is not what the writer requires. 
It just looks like a workaround as the tests will produce random input data, but OPENVINO has no limitation here.  

And another function normalize_center() just converts [x, y, width, height] to [y1, x1, y2, x2], they are different expressions of the same box, so keep it.

Reply:
Do I understand correctly that this just changes the semantics of the box coordinates? And specifically, with this commented out, boxes provided in [y1, x1, y2, x2] order will be handled correctly and boxes in [y1, x2, y2, x1] order will be handled incorrectly? And, conversely, in the original form, boxes in [y1, x1, y2, x2] order will be handled incorrectly while boxes in [y1, x2, y2, x1] order will be handled correctly? Also, does commenting this out make the behavior of the reference implementation match the order specified in the docs?

If that understanding is correct, then we should make the change but also add a comment like `// TODO: Check if docs or original reference implementation is correct` so we know to come back to this during upstreaming (because it's theoretically possible the reference impl. is correct and the docs are wrong).

Or let me know if I misunderstood.

Reply:
Thanks, have added comment here. 
From https://docs.openvinotoolkit.org/latest/openvino_docs_ops_sort_NonMaxSuppression_5.html, the order shall always be [y1, x1, y2, x2] in CORNER mode, but the function want to make it to [ymin, xmin, ymax, xmax], so the data value changes our behaviors here. And [y1, x2, y2, x1] is always wrong, as even if we pass the box info in this order, it  may be changed.  So if we keep the function, some boxes are right and some boxes are wrong here. 
We may recheck this when we back to upstream as the docs or ref may change that time.

Reply:
Add https://github.com/plaidml/plaidml/pull/1757 to change box info to [ymin, xmin, ymax, xmax] in BoxesDecodeMode::NMS mode. This PR only do sort now.

Reply:
Sounds good, thanks!